frontend/src/types.ts

```diff
@@ -818,6 +818,18 @@ export interface SessionRecordingSnapshotSource {
     blob_key?: string
 }
 
+export type SessionRecordingSnapshotParams =
```

Comment:
Could we reuse this type on `getBlobSnapshots` method, rather than the more generic `Record<string, any>` we have right now